docs/glossary.rst

```diff
@@ -313,8 +313,9 @@ towards particular customer use cases: lower capacity vs. higher capacity, more
 storage vs. more processing power, and so forth. They can also be further
 adjusted for different :ref:`scale units<glossary-scale-unit>` per plan.
 Currently there are four subscription plans available on our cloud offers, with
-more to come in the near future. For more information on choosing the right
-subscription plan, refer to our documentation `on the subject`_.
+more to come in the near future, as well as a contract offer. For more
+information on choosing the right subscription plan, refer to our documentation
+`on the subject`_.
```

Comment:
oops sorry just caught this, but "on the subject" is another candidate for being reworded as a noun phrase :)